src/main/java/it/gov/pagopa/pu/migration/service/migration/MigrationProcessingService.java

```diff
@@ -0,0 +1,65 @@
+package it.gov.pagopa.pu.migration.service.migration;
+
+import com.opencsv.exceptions.CsvException;
+import it.gov.pagopa.pu.migration.service.file.ErrorArchiverService;
+
+import java.util.Iterator;
+import java.util.List;
+
+public abstract class MigrationProcessingService<T, R, E extends it.gov.pagopa.pu.migration.dto.ErrorFileDTO> {
```

Comment:
let's use import instead of fullpackage definition
```suggestion
public abstract class MigrationProcessingService<T, R, E extends ErrorFileDTO> {
```